de.dentrassi.pm.maven/src/de/dentrassi/pm/maven/internal/MavenRepositoryChannelAggregator.java

```diff
@@ -174,76 +185,92 @@ private String makeUrl ( final String channelId )
         return result;
     }
 
-    private MavenInformation makeInfo ( final ArtifactInformation art, final Map<String, ArtifactInformation> map )
+    private Collection<MavenInformation> getInfos ( final ArtifactInformation art, final Map<String, ArtifactInformation> map )
     {
-        final MavenInformation info = new MavenInformation ();
+        final Collection<MavenInformation> infos = new LinkedList<> ();
+
+        /*
+         * Try to use the info of the artifact them-self.
+         */
 
         try
         {
+            final MavenInformation info = new MavenInformation ();
             MetaKeys.bind ( info, art.getMetaData () );
+            if ( info.getGroupId () != null & info.getArtifactId () != null && info.getVersion () != null )
+            {
+                // found direct meta data
+                infos.add ( info );
```

Comment:
Well direct meta data will be present if there is provided meta data present. For example the maven import or the `mvn deploy` processes both add direct metadata.